packages/math-input/src/components/keypad/mobile-keypad.tsx

```diff
@@ -158,12 +163,18 @@ class MobileKeypad extends React.Component<Props, State> implements KeypadAPI {
 
     render(): React.ReactNode {
         const {style} = this.props;
-        const {active, containerWidth, cursor, keypadConfig} = this.state;
+        const {active, hasBeenActivated, containerWidth, cursor, keypadConfig} =
+            this.state;
 
         const containerStyle = [
             // internal styles
             styles.keypadContainer,
             active && styles.activeKeypadContainer,
+            // If the keypad is yet to have ever been activated, we keep it invisible
```

Comment:
```suggestion
            // If the keypad is yet to have never been activated, we keep it invisible
```

Reply:
I just copy/pasted and the wording is a little weird, but I think the original is more correct.

Reply:
Oooooh, it was my misreading!! 🙃